backend/tests/test_integration_schwab.py

```
@@ -75,15 +75,17 @@ def _insert_tokens(session_local_fn, access_minutes=30, refresh_days=7):
 class TestSettingsEndpointSchwab:
     """GET /api/settings — schwab_configured and schwab_token_expires fields."""
 
-    def test_unconfigured_returns_false(self, client, test_session_local):
+    @pytest.mark.integration
+    def test_unconfigured_returns_false(self, client, session_local):
```

Ruff (0.15.13)

[warning] Unused method argument: `session_local`

(ARG002)